bootstrap.sh

```diff
@@ -33,7 +33,7 @@ function installPackages() {
 
   # Install firefox developer edition
   brew tap homebrew/cask-versions
-  brew install --cask firefox-developer-edition
+  brew install --cask firefox@developer-edition
```

Comment:
thank you for this update. i didn't know they update it like this. since they supports from brew directly, i don't think we need `brew tap homebrew/cask-versions`. let's remove that line.